app/renderer/components/tooltip.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+module.exports = {
+    props: ['color'],
+    template: `
+    <div class="tooltip">
+        <div class="tooltip-text">
+            {{ status }}
+        </div>
+    </div>
+    `,
+    computed: {
+        status() {
+            switch (this.color) {
+                case "black":
+                    return "Stopped";
+                case "red":
+                    return "Error";
+                case "blue":
+                    return "Running";
+                case "green":
+                    return "OK";
+                case "grey":
+                default:
+                    return "N/A";
```

Comment:
In the default case, the tooltip shouldn't be shown. There is no grey color afaik